tests/AdagucTests/TestWMSTimeHeightProfiles.py

```diff
@@ -0,0 +1,173 @@
+            env=self.env,
+        )
+        AdagucTestTools().writetofile(self.testresultspath + filename, data.getvalue())
+        self.assertEqual(status, 0)
+        self.assertTrue(
+            AdagucTestTools().compareGetCapabilitiesXML(
+                self.testresultspath + filename, self.expectedoutputsspath + filename
+            )
+        )
+
+    def test_wms_getmap_timeheightprofiles(self):
+        """
+        Check if WMS GetMap on time height profiles is as expected
+        """
+        AdagucTestTools().cleanTempDir()
+        filename = "test_WMSGetMap_TimeHeightProfiles_4326_beta_raw.png"
+        wms_arg = "source=test/ceilonet/ceilonet_chm15k_backsct_la1_t12s_v1.0_06310_A20231202_extracted_small.nc&SERVICE=WMS&VERSION=1.3.0&REQUEST=GetMap&LAYERS=beta_raw&WIDTH=256&HEIGHT=256&CRS=EPSG%3A3857&BBOX=259793.6688373399,6178346.966712794,1082625.1470173844,7357680.642536524&STYLES=auto%2Fnearest&FORMAT=image/png&TRANSPARENT=TRUE&&time=2023-12-02T06%3A40%3A00Z"
+        status, data, _ = AdagucTestTools().runADAGUCServer(wms_arg, env=self.env)
+        AdagucTestTools().writetofile(self.testresultspath + filename, data.getvalue())
+        self.assertEqual(status, 0)
+        self.assertEqual(
+            data.getvalue(),
+            AdagucTestTools().readfromfile(self.expectedoutputsspath + filename),
+            self.expectedoutputsspath + filename,
+        )
+
+    def test_wmsgetfeatureinfo_timeheightprofiles(self):
+        """
+        Check if WMS GetFeatureInfo on time height profiles is as expected
+        """
+        AdagucTestTools().cleanTempDir()
+        filename = "test_WMSGetFeatureInfoAsPng_TimeHeightProfiles.png"
+        wms_arg = "source=test/ceilonet/ceilonet_chm15k_backsct_la1_t12s_v1.0_06310_A20231202_extracted_small.nc&SERVICE=WMS&REQUEST=GetFeatureInfo&VERSION=1.3.0&LAYERS=beta_raw&QUERY_LAYERS=beta_raw&BBOX=29109.947643979103,6500000,1190890.052356021,7200000&CRS=EPSG%3A3857&WIDTH=128&HEIGHT=512&I=707&J=557&FORMAT=image/gif&INFO_FORMAT=image/png&STYLES=&time=2023-12-02T06:40:00Z/2023-12-02T06:45:00Z&elevation=0/2500"
+        # TODO: Check why we first need to do a getcapabilities call?
+        status, data, _ = AdagucTestTools().runADAGUCServer(
+            "source=test/ceilonet/ceilonet_chm15k_backsct_la1_t12s_v1.0_06310_A20231202_extracted_small.nc&SERVICE=WMS&request=getcapabilities",
+            env=self.env,
+        )
+        status, data, _ = AdagucTestTools().runADAGUCServer(wms_arg, env=self.env)
+        AdagucTestTools().writetofile(self.testresultspath + filename, data.getvalue())
+        self.assertEqual(status, 0)
+        self.assertEqual(
+            data.getvalue(),
+            AdagucTestTools().readfromfile(self.expectedoutputsspath + filename),
+            self.expectedoutputsspath + filename,
+        )
+
+    def test_wmsgetfeatureinfo_png_timeheightprofiles_as_dataset(self):
```

Comment:
It is not clear to me how this test relates to the GetCapabilities test above.

Reply:
Tried to make it clear in the docstring, this test uses a dataset configuration via `dataset=`, the previous uses a single file via `source=`